langlib/lang.array/src/main/ballerina/array.bal

```diff
@@ -70,6 +88,33 @@ public isolated function enumerate(Type[] arr) returns [int, Type][] = @java:Met
 
 # Applies a function to each member of an array and returns an array of the results.
 #
+# ```ballerina
+# string[] greetings = ["Hello", "Bonjour", "Hola", "Ciao"];
+# int[] lengths = greetings.'map((greeting) => greeting.length());
+#
```

Comment:
Would it be better if a comment is added here to distinguish between the two examples?

Reply:
IMO, it is self-explanatory, since both calls use the same number of arguments unlike with something like `array:sort`.